packages/beacon-node/src/chain/stateCache/stateContextCache.ts

```diff
@@ -6,7 +6,7 @@ import {IMetrics} from "../../metrics/index.js";
 import {MapTracker} from "./mapMetrics.js";
 import {stateInternalCachePopulated} from "./stateContextCheckpointsCache.js";
 
-const MAX_STATES = 3 * 32;
+const MAX_STATES = 6 * 32;
```

Comment:
we need to investigate the impact of this with regards to the heap memory (which lead to `gc` percentage metrics). In the past when chain was not finalized for a while I saw the heap memory spiked at the same time.